po/tr.po

```diff
@@ -808,7 +818,7 @@ msgstr "Hata veya fikir bildirin"
 #. per line), they will be displayed in the "about" dialog
 #: src/main.py
 msgid "translator-credits"
-msgstr "Serdar Sağlam <teknomobil@yandex.com>"
+msgstr "çevirmenler-Serdar SAĞLAM-Serkan ÖNDER"
```

Comment:
when there are several translators, the convention is to separate each credit with a line break (`\n`), as explained in the comment above

there is no need to write "çevirmenler": these lines will be shown in the about dialog, next to the native GTK "Çevirmenler" label. Also, please don't remove the mail address of the previous translator, what he did is a correct and standard crediting format, automatically rendered as a clickable `mailto:` link in the credit list.